From e4974594c5c3147673941834f73ae3a9ce6ffa1b Mon Sep 17 00:00:00 2001
From: Jami Kettunen <jami.kettunen@protonmail.com>
Date: Wed, 16 Sep 2020 11:01:07 +0300
Subject: [PATCH] droidmedia: Fix build since android-9.0.0_r60

Upstream removed the const qualifier from getLayerDebugInfo() which
breaks our build if we don't update this as well.

Reference:
https://github.com/LineageOS/android_frameworks_native/commit/23a8ea7cd2aa38cc28d37ae0e1e06043ec5da0b3
---
 services/services_9_0_0.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/services_9_0_0.h b/services/services_9_0_0.h
index 85a942e..69d7936 100644
--- a/services/services_9_0_0.h
+++ b/services/services_9_0_0.h
@@ -161,7 +161,7 @@ class MiniSurfaceFlinger : public BinderService<MiniSurfaceFlinger>,
         return BAD_VALUE;
     }
 
-    status_t getLayerDebugInfo(std::vector<LayerDebugInfo>*) const {
+    status_t getLayerDebugInfo(std::vector<LayerDebugInfo>*) {
         return 0;
     }
 };
